scripts/composer-update.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+cd "$DRUPAL_TI_DRUPAL_DIR"
+cd core
+composer drupal-rebuild
+composer update embed --prefer-source
```

Comment:
Sometimes travis fails to download packages from Github, due to API limit or whatever. This option helps avoid that. One issue which I referenced: https://github.com/composer/composer/issues/1314